projects/shikshalokam/src/lib/core-module/components/headers/headers.component.scss

```diff
@@ -0,0 +1,26 @@
+mat-sidenav {
```

Comment:
use of this class??

Reply:
we are over riding this css with our custom width